packages/react-frontend/src/pages/Login.jsx

```diff
@@ -0,0 +1,135 @@
+import { React, useState, useEffect } from "react";
+
+function LoginForm(props) {
+  const [creds, setCreds] = useState({
+    username: "",
+    password: "",
+  });
+
+  return (
+    <form>
+      <label htmlFor="username">UserName</label>
+      <input
+        type="text"
+        name="username"
+        id="username"
+        value={creds.username}
+        onChange={handleChange}
+      />
+      <label htmlFor="password">Password</label>
+      <input
+        type="password"
+        name="password"
+        id="password"
+        value={creds.password}
+        onChange={handleChange}
+      />
+      <input
+        type="button"
+        value={props.buttonLabel || "Log In"}
+        onClick={submitForm}
+      />
+    </form>
+  );
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    switch (name) {
+      case "username":
+        setCreds({ ...creds, username: value });
+        break;
+      case "password":
+        setCreds({ ...creds, password: value });
+        break;
+    }
+  }
+
+  function submitForm() {
+    props.handleSubmit(creds);
+    setCreds({ username: "", password: "" });
+  }
+}
+
+function LoginPage() {
+  const INVALID_TOKEN = "INVALID_TOKEN";
+  const [token, setToken] = useState(localStorage.getItem("authToken") || INVALID_TOKEN);
+  //const [token, setToken] = useState(INVALID_TOKEN);
+  const [message, setMessage] = useState("");
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+  const API_PREFIX = "http://localhost:8000";
```

Comment:
We are going to have to change this to
const API_PREFIX = https://songsmith.azurewebsites.net